src/Support/Enums/Alignment.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace OwowAgency\Gossip\Support\Enums;
+
+class Alignment
+{
+    public const LEADING = 'leading';
+
+    public const TRAILING = 'trailing';
```

Comment:
Are these the common terms in alignment of chat messages?

Reply:
@Obbut will haunt you with his katana if we change this to `left` and `right`..